.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "arcanis.vscode-zipfs",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode"
+  ]
+}
```

Comment:
When prompted by vscode to install these extensions you need to install them otherwise code lookup and other stuff just doesn't work.